--- slackpkg.SlackBuild	2024-07-20 20:09:32.800185690 +0300
+++ slackpkg.SlackBuild.new	2025-03-17 09:24:19.000000000 +0200
@@ -25,7 +25,7 @@
 PKGNAM=slackpkg
 VERSION=${VERSION:-15.0.10}
 ARCH="noarch"
-BUILD=${BUILD:-4}
+BUILD=${BUILD:-1}
 
 # If the variable PRINT_PACKAGE_NAME is set, then this script will report what
 # the name of the created package would be, and then exit. This information
@@ -55,7 +55,7 @@
 
 # Prepare /etc directory:
 mkdir -pv $PKG/etc/slackpkg
-cp -av blacklist.new slackpkg.conf.new post-functions.conf-sample \
+cp -av blacklist.new whitelist.new slackpkg.conf.new post-functions.conf-sample \
   $PKG/etc/slackpkg
 chmod 644 $PKG/etc/slackpkg/*
 chown root:root $PKG/etc/slackpkg/*
